repo-stats/dependencies_state/get_data_on_dependencies.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+"""
+Python script to output state of each dependency for a repo.
+
+Currently only gets data from pip show for each dependency and outputs it in csv file
+"""
+
+import argparse
+from latest_state import LatestState
+from current_state import CurrentState
+
+
+def auto_get_data(arguments):
+    """
+    use this function for the first time you are in a given env, this will get info on all packages installed
+    and on all the latest packages avaible
+    """
+    #TODO(jinder): figure out how to do latest vs current env things
+    Namespace(auto_script=False, csv_path=None, latest=False, read_json_file=None, reget_data=False, save_raw_data=None)
```

Comment:
Is this function a work in progress?  Don't see a declaration for `Namespace`.

Reply:
No, this function is done, just didn't remove old comment and comment out namespace(the namespace is for what is inside arguments)